nix/overlay.nix

```diff
@@ -0,0 +1,27 @@
+final: prev: {
+  ocaml-ng = (prev.lib.mapAttrs (_: ocamlVersion:
+    ocamlVersion.overrideScope' (oself: osuper: {
+      pollinate = (final.callPackage ./default.nix {
+        ocamlPackages = osuper;
+        doCheck = true;
+      });
+    })) (builtins.removeAttrs prev.ocaml-ng [
```

Comment:
Looks like it is exactly the same than in `flake.nix`, can we make it using the same to avoid duplication?

Reply:
Yes, but I was lazy